mandatory/2-function-creation.js

```diff
@@ -6,7 +6,12 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (typeof num === "string") {return false}
+  else if (num===100) {return true}
+  else if (num%2==0 & num <= 100 ) {return true} 
+  return false
```

Comment:
you could concatenate the statements with an `&&` and omit the true to make it more compact.

Reply:
yes, changed it to &&

Reply:
that's cool. Line 11 is not redundant as it is included in line 12 <**=**

Reply:
also usually when you have statement that return booleans you can write them in a compact form.
For example, if I had:
```
function(banana){
   if( banana == true) {
   return false
  }
}
```
I could just go for:
```


```
function(banana){
   return !banana
}
```
if the statement is true, it is even simpler cause I don't need the bang `!`